cmd/spire-server/cli/entry/util.go

```diff
@@ -91,6 +95,34 @@ func printEntry(e *common.RegistrationEntry) {
 	fmt.Println()
 }
 
+// ParseFile parses JSON represented RegistrationEntries
+// if path is "" or "*" read JSON from STDIN
+func ParseFile(in io.Reader, path string) ([]*common.RegistrationEntry, error) {
+	entries := &common.RegistrationEntries{}
+
+	var r io.Reader
+	if path == "-" {
+		r = in
```

Comment:
this could be simplified to remove the `else` with something like:
```
r := stdin
if path != "-" {
	f, err := os.Open(path)
	if err != nil {
		return nil, err
	}
	defer f.Close()
	r = f
}
```

Reply:
6096a68